tests/test_root.py

```diff
@@ -148,54 +151,12 @@ def test_root_gen_sign_verify():
     )
     rmd = signing.wrap_as_signable(rmd)
 
-    signed_portion = rmd["signed"]
+    # Sign it with the GPG key.
+    root_signing.sign_root_metadata_dict_via_gpg(rmd, SAMPLE_FINGERPRINT)
```

Comment:
Looks like a typo?  This is importing `sign_root_metadata_via_gpg` and then calling `sign_root_metadata_dict_via_gpg`.  Separately, it's also not using the reference imported in line 18, but instead just calling it from the `root_signing` module.